pkg/controller/controller.go

```diff
@@ -41,23 +40,35 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
 
 	rbacsyncv1alpha "github.com/cruise-automation/rbacsync/pkg/apis/rbacsync/v1alpha"
 	clientset "github.com/cruise-automation/rbacsync/pkg/generated/clientset/versioned"
 	rbacsyncscheme "github.com/cruise-automation/rbacsync/pkg/generated/clientset/versioned/scheme"
 	informers "github.com/cruise-automation/rbacsync/pkg/generated/informers/externalversions/rbacsync/v1alpha"
 	listers "github.com/cruise-automation/rbacsync/pkg/generated/listers/rbacsync/v1alpha"
 	"github.com/cruise-automation/rbacsync/pkg/groups"
+	"github.com/cruise-automation/rbacsync/pkg/metrics"
 )
 
 const (
 	controllerAgentName = "rbacsync"
 )
 
+const (
+	MetricsRBACSyncConfig        = "RBACSyncConfig"
+	MetricsRBACSyncClusterConfig = "RBACSyncClusterConfig"
```

Comment:
Could you make this consistent with the other references? ClusterRBACSyncConfig is what I think we use.

Reply:
will do!